frontend/src/pages/audio/_id/index.vue

```diff
@@ -1,32 +1,41 @@
 <template>
-  <main :id="skipToContentTargetId" tabindex="-1">
-    <div v-if="backToSearchPath" class="w-full px-2 py-2 md:px-6">
-      <VBackToSearchResultsLink
-        :id="$route.params.id"
-        :href="backToSearchPath"
-      />
-    </div>
-
-    <template v-if="audio">
-      <VAudioTrack
-        layout="full"
-        :audio="audio"
-        class="main-track"
-        @interacted="sendAudioEvent($event, 'AudioDetailPage')"
-      />
-      <div
-        class="mx-auto mt-10 flex flex-col gap-10 px-6 lg:mt-16 lg:max-w-5xl lg:gap-16"
-      >
-        <VMediaReuse data-testid="audio-attribution" :media="audio" />
-        <VAudioDetails data-testid="audio-info" :audio="audio" />
-        <VRelatedAudio
-          v-if="relatedMedia.length || relatedFetchState.isFetching"
-          :media="relatedMedia"
-          :fetch-state="relatedFetchState"
-          @interacted="sendAudioEvent($event, 'VRelatedAudio')"
+  <main :id="skipToContentTargetId" tabindex="-1" class="relative flex-grow">
+    <template v-if="visibility !== 'sensitive-hidden'">
+      <div class="flex w-full justify-between px-2 py-2 md:px-6">
```

Comment:
This div should probably moved out of the `visibility !== 'sensitive-hidden'` conditional template, otherwise the non-sensitive audio pages won't have the back to search results button.